pkgs/tools/misc/cht.sh/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, fetchFromGitHub
+, makeWrapper
+, curl
+, ncurses
+, rlwrap
+, xsel
+}:
+
+stdenv.mkDerivation rec {
+  name = "cht.sh-${version}";
+  version = "unstable-2018-11-02";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  src = fetchFromGitHub {
+    owner = "chubin";
+    repo = "cheat.sh";
+    rev = "9595805ac68b3c096f7c51fa024dcb97a7dfac44";
+    sha256 = "11g8say5fksg0zg0bqrgl92rprn4lwp20g9rz1i0r38f0jy3nyrf";
+  };
+
+  installPhase = ''
+    install -m755 -D share/cht.sh.txt "$out/bin/cht.sh"
+    wrapProgram "$out/bin/cht.sh" --prefix PATH : \
+    "${stdenv.lib.makeBinPath [ curl rlwrap ncurses xsel ]}"
```

Comment:
It's customary to indent the second part of a command following a backslash.

You can do:
```nix
  wrapProgram $out/bin/cht.sh --prefix PATH : \
    "${stdenv.lib.makeBinPath [ curl rlwrap ncurses xsel ]}"
```

Or:
```nix
  wrapProgram $out/bin/cht.sh \
    --prefix PATH : "${stdenv.lib.makeBinPath [ curl rlwrap ncurses xsel ]}"
```

Reply:
Good catch! Fixed